hazelcast/src/test/java/com/hazelcast/multimap/MultiMapTest.java

```diff
@@ -104,6 +114,184 @@ public void testMultiMapPutGetRemove() {
         assertEquals(0, multiMap.size());
     }
 
+    protected <K, V> EntryListener<K, V> putAllEntryListenerBuilder(Consumer<EntryEvent<K, V>> f) {
+        return new EntryAdapter<K, V>() {
+            public void entryAdded(EntryEvent<K, V> event) {
+                f.accept(event);
+            }
+
+            public void entryRemoved(EntryEvent<K, V> event) {
+                System.out.println("entryRemoved " + event);
+            }
+
+            public void entryEvicted(EntryEvent<K, V> event) {
+                entryRemoved(event);
+            }
+
+            @Override
+            public void mapEvicted(MapEvent event) {
+            }
+
+            @Override
+            public void mapCleared(MapEvent event) {
+                System.out.println("mapCleared " + event);
+            }
+        };
+    }
+
+    protected HazelcastInstance testMultiMapPutAllSetup() {
+        MultiMapConfig multiMapConfig1 = new MultiMapConfig()
+                .setName("testMultiMapPutAllMapList")
+                .setValueCollectionType("LIST")
```

Comment:
you can use the compile-time safe variant `setValueCollectionType(ValueCollectionType valueCollectionType)` (here + 3 more instances below).